### ISSUE TITLE
Braces not formatted with switches containing tuples, extensions, or "required init" function.

### ISSUE THREAD

**Issue by PompeiaPaetenari** [nicklockwood/SwiftFormat#574]
So sorry to bother you again about this, but switch statements with tuples are still not happy (in either direction to allman or back):

```swift
switch (firstOctet, secondOctet, thirdOctet, fourthOctet) {
	case (0, 0, 0, 0):
		return .failure(.unableToParseIP)

	case (10, 92, _, _):
		switch thirdOctet
		{
```
Got ^ after running with allman true.

```swift
  switch (firstOctet, secondOctet, thirdOctet, fourthOctet)
  {
  case (0, 0, 0, 0):
    return .failure(.unableToParseIP)

  case (10, 92, _, _):
    switch thirdOctet {
```
And ^ when I move the brace manually and run without `--allman`

Similarly for extensions when converting to/from allman style:
```swift
extension ViewModel {
	struct ShareOption: Equatable
	{
		let iconName: String
		let label: String
		let action: ShareViewModel.Action
		let accessibilityIdentifier: String
	}
}
```

Also required init:
```swift
	required init?(coder _: NSCoder) {
		fatalError("init(coder:) has not been implemented")
	}
```
which does not get formatted in either direction (move the brace down and run without `--allman` and it does not move back).

Currently using `0.44.4` as installed from Homebrew.

**Comment by nicklockwood** [comment]
@PompeiaPaetenari thanks for reporting this, I'll get it fixed for the next release.

**Comment by nicklockwood** [comment]
@PompeiaPaetenari fixed in 0.44.5
